templates/ovndbcluster/bin/setup.sh

```diff
@@ -27,7 +27,7 @@ if [[ "$(hostname)" != "{{ .SERVICE_NAME }}-0" ]]; then
     #ovsdb-tool join-cluster /etc/ovn/ovn${DB_TYPE}_db.db ${DB_NAME} tcp:$(hostname).{{ .SERVICE_NAME }}.openstack.svc.cluster.local:${RAFT_PORT} tcp:{{ .SERVICE_NAME }}-0.{{ .SERVICE_NAME }}.openstack.svc.cluster.local:${RAFT_PORT}
     OPTS="--db-${DB_TYPE}-cluster-remote-proto=tcp --db-${DB_TYPE}-cluster-remote-addr={{ .SERVICE_NAME }}-0.{{ .SERVICE_NAME }}.openstack.svc.cluster.local --db-${DB_TYPE}-cluster-remote-port=${RAFT_PORT}"
 fi
-/usr/local/bin/start-${DB_TYPE}-db-server --db-${DB_TYPE}-create-insecure-remote=yes --db-${DB_TYPE}-election-timer={{ .OVN_ELECTION_TIMER }} --db-${DB_TYPE}-cluster-local-proto=tcp \
```

Comment:
removal of this isn't mentioned in the commit message. Should it be removed?

Reply:
@slawqo yes it's required to allow setting connections, basically just removed --db-${DB_TYPE}-create-insecure-remote=yes flag

Reply:
ok